requirements.txt

```diff
@@ -1,12 +1,12 @@
-Sphinx==1.6.5
-dj-database-url==0.4.2
-flake8==2.4.0
+dj-database-url==0.5.0
```

Comment:
Might be worth switching this project to `poetry` soon to reduce these files:
- `requirements.txt`
- `setup.py`
- `requirements.test.txt`
- `install_requires.txt`
Down to just:
- `pyproject.toml` 

The project should still be buildable by `pip`/`pipenv`/etc as a dependency without poetry